pkg/config/config.go

```diff
@@ -228,17 +230,66 @@ func (vcConfig *VCConfig) validate() error {
 	return nil
 }
 
+func removeEmptyItem(source []string) []string {
```

Comment:
this code is just from https://github.com/vmware-tanzu/nsx-operator/pull/251
why not pick PR-251 to VPC_dev firstly, and then debase your patch?

Reply:
The change is simple. so copy is easier than cherry-pick.